src/common/utils.lua

```diff
@@ -348,6 +348,13 @@ function utils.createHandler(tagName, tagValue, handler, position)
 				notices.notifyState(msg, AntRegistryId)
 			end
 
+			-- send a patch notice on any action that changes the state
```

Comment:
Is it critical to send this to the device AFTER sending the result notice? I'm wondering whether there's tradeoffs and/or a logical reason to have the outbox have these in a specific order. I'm not even sure there's a spec on how outbox ordering could impact the system in the general sense...

Reply:
it is not critical for it to be after - but if there is an invalid message (bad tags, improper formatting, or too big) then it is my understanding the MU will blow up and not crank/send ANY messages.